src/content/profile-data.js

```diff
@@ -735,34 +735,36 @@ export function getSampleIndexClosestToTime(samples: SamplesTable, time: number)
   return samples.length - 1;
 }
 
-export function getJankInstances(samples: SamplesTable, processType: string, thresholdInMs: number): TracingMarker[] {
+function addTracingMarker(tracingMarkers: TracingMarker[], start, dur, title, name, data) {
+  tracingMarkers.push({start, dur, title, name, data});
+}
+
+export function getJankInstances(samples: SamplesTable, threadName: string, thresholdInMs: number): TracingMarker[] {
   let lastResponsiveness = 0;
   let lastTimestamp = 0;
   const jankInstances = [];
   for (let i = 0; i < samples.length; i++) {
     const currentResponsiveness = samples.responsiveness[i];
     if (currentResponsiveness < lastResponsiveness) {
       if (lastResponsiveness >= thresholdInMs) {
-        jankInstances.push({
-          start: lastTimestamp - lastResponsiveness,
-          dur: lastResponsiveness,
-          title: `${lastResponsiveness.toFixed(2)}ms event processing delay on ${processType} thread`,
-          name: 'Jank',
-          data: null,
-        });
+        addTracingMarker(jankInstances,
```

Comment:
I know in the issue, @julienw mentioned moving this out into a new function, but I think this makes it less readable, and I'm not seeing how this adds value. Flow will let us know if we are incorrectly formatting the TracingMarker. Plus the new function isn't pure and operates on a side effect.